src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkBookingService.kt

```diff
@@ -294,4 +294,14 @@ class VideoLinkBookingService(
       prisonApiService.getLocation(it) ?: throw ValidationException("$prefix locationId $it not found in NOMIS.")
     }
   }
+
+  fun deleteAppointments(appointmentId: Long) {
+    val videoLinkAppointments = videoLinkAppointmentRepository.findAllByAppointmentId(appointmentId)
```

Comment:
this will need to look all appointments for the booking that is associated with the appointment

Reply:
it is sorted by deleting whole booking.